scripts/conversionScripts/conversion_from_old_hdf5_to_new/reader_old_HDF5_format.py

```diff
@@ -0,0 +1,482 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Created on Mar 25, 2013
+
+@author: alfoa
+"""
```

Comment:
Maybe we should add more header to this module explaining it's the old database (and note that old is from 2013-2018 rework?).

Reply:
added better explanations